src/views/OrderDetail.vue

```diff
@@ -120,7 +120,7 @@ export default defineComponent({
     })
   },
   ionViewDidEnter() {
-    if(this.order.items) this.order.items.map((item) => item['reason'] = this.unfillableReason[0].id);
+    if(this.orderitems) this.order.items.map((item) => item['reason'] = this.unfillableReason[0].id);
```

Comment:
Unwanted changes. I think this is wrong variable name. Please revert.

Reply:
Reverted unintentional change.